source/git-util.js

```diff
@@ -41,7 +41,15 @@ exports.previousTagOrFirstCommit = async () => {
 		return firstCommit();
 	}
 
-	return tags[tags.length - 2];
+	try {
+		// Return the tag before the latest one.
+		const latest = await exports.latestTag();
+		const index = tags.indexOf(latest);
```

Comment:
Use `Array#find`.

Reply:
Ok, can you tell me why? (just to know 😊)

Reply:
Because it's IMHO more readable.